Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkHierarchyRootComponent.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TransformBus.h>
+#include <Source/AutoGen/NetworkHierarchyRootComponent.AutoComponent.h>
+#include <Multiplayer/Components/NetworkHierarchyBus.h>
+
+namespace Multiplayer
+{
+    //! @class NetworkHierarchyRootComponent
+    //! @brief Component that declares the top level entity of a network hierarchy.
+    /*
+     * Call @GetHierarchyChildren to get the list of hierarchical entities.
```

Comment:
I dont see the function called GetHierarchyChildren in the class below. Was this meant to say "GetHierarchicalEntities"?

Reply:
Yep, good catch, I've refactored several times and didn't update the comments. I'll fix it up.